test/test.js

```diff
@@ -0,0 +1,5 @@
+//remove falsey values from data
```

Comment:
is there a better name than test.js?

Reply:
Maybe newUnitTest.js?  It's there to avoid running all tests, just the one currently being worked on.  I'll remove it at the end.